components/raftstore/src/store/fsm/peer.rs

```diff
@@ -101,6 +102,18 @@ pub struct PeerFsm<E: KvEngine> {
     pub receiver: Receiver<PeerMsg<E>>,
     /// when snapshot is generating or sending, skip split check at most REGION_SPLIT_SKIT_MAX_COUNT times.
     skip_split_count: usize,
+
+    // Batch raft command which has the same header into an entry
+    batch_req_builder: BatchRaftCmdRequestBuilder,
+}
+
+const MAX_BATCH_KEY_NUM: usize = 128;
```

Comment:
Redundant const definition.

Reply:
Why? Shall I use  a magic number directly in `should_finish`?

Reply:
The const defined in apply and peer should be unified.